Makefile

```diff
@@ -1029,6 +1022,7 @@ dist/test/c/%.c: $(ALL_KRML_FILES)
 	  -library Hacl.P256,Hacl.K256.*,Hacl.Impl.*,EverCrypt.* \
 	  -fparentheses -fcurly-braces -fno-shadow \
 	  -minimal -add-include '"krmllib.h"' \
+	  -add-include '"libintvector.h"' \
```

Comment:
I imagine that, since this is only for dist/test/, that's not much of a problem, but why is it necessary to always include the libintvector header?

Reply:
this is because the test contains a copy of the struct definition for the evercrypt hash internal state, which contains a blake2b_256 state, which refers to the type Lib_IntVector_Vec256, so this means that this type needs to be in scope for the test to compile

Reply:
We discussed this offline, this addition is actually related to another PR, which adds Blake2 to Evercrypt's hashes and requires libintvector.
The problem here is that this additional header is now included in all C tests generated from F*, including for instance CSHAKE or ECDSA.
However, since this is only for test purposes, we think it's fine to keep the makefile simple and avoid special-casing for hash tests.